lib/conjur/policy/yaml/handler.rb

```diff
@@ -229,6 +229,11 @@ def alias anchor
   
           # Begins a new map entry.
           def scalar value, tag, quoted, anchor
+            if @existing_members.include?(value)
+              raise "Duplicate attribute of type #{value}"
```

Comment:
"of type" reads oddly here, lets change this to something like:

```suggestion
              raise "Duplicate attribute: '#{value}'"
```